surveys/2024-04/questions.md

```diff
@@ -0,0 +1,142 @@
+# Survey April 2024
+
+## Demographics
+
+- Where are you from? (countries list if possible - or open question)
+    - country
+
+- How long have you been working with Node.js? (numbers list from 0 to 10+)
+
+- In what kind of organizations are you working in?
+    - big tech
+    - company
+    - startup
+    - university
+    - individual
+    - other
+
+- Are you part/following some Working Groups? Are you involved in some Node.js community  Working Groups ? If yes which one (open question)
```

Comment:
```suggestion
- Are you part/following some Working Groups? Are you involved in some Node.js community Working Groups ? If yes which one (open question)
```

Not sure if it matters but there was a double space that should be a single space